addons/wowl/static/src/views/legacy/legacy.ts

```diff
@@ -250,3 +250,31 @@ odoo.define("wowl.legacyViews", async function (require: any) {
   }
   legacyViewRegistry.onAdd(registerView);
 });
+
+odoo.define("wowl.legacySystrayMenuItems", function (require: any) {
+  require("wowl.legacySetup");
+  const { ComponentAdapter } = require("web.OwlCompatibility");
+  const legacySystrayMenu = require("web.SystrayMenu");
+
+  class SystrayItemAdapter extends ComponentAdapter {
+    env = Component.env;
+  }
+
+  const legacySystrayMenuItems = legacySystrayMenu.Items as any[];
+  // registers the legacy systray menu items from the legacy systray registry
+  // to the wowl one, but wrapped into Owl components
+  legacySystrayMenuItems.forEach((item, index) => {
+    if (item.prototype.template !== "UserMenu") {
```

Comment:
A small suggestion to make it super explicit:
```suggestion
  // registers the legacy systray menu items from the legacy systray registry
  // to the wowl one, but wrapped into Owl components
  legacySystrayMenuItems.forEach((item, index) => {
    // blacklisting already wowl converted items
    const blacklist = ["UserMenu"];
    if (!blacklist.includes(item.prototype.template)) {
```